spk/he853/src/installer.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# Package
+PACKAGE="he853"
+DNAME="HomeEasy HE853 USB device executable"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/${PACKAGE} /usr/local/bin/${PACKAGE}
```

Comment:
If you add a symlink, be sure to remove it during postuninst. Although I'm not sure if there's added value in putting the binary on PATH at all..